modules/ROOT/pages/enable_apps_deployed_to_cloud.adoc

```diff
@@ -0,0 +1,15 @@
+= Enable Mule Applications Deployed to CloudHub
```

Comment:
We need to divide this up a bit I think.  There are three ways to enable.

1.  Via the UI (what you have)
2. By adding this property into the properties tab in the CloudHub settings page:

anypoint.platform.config.analytics.agent.enabled=true

3.  For CI/CD customers, they can do it by setting this property during their CI/CD deployment via the Anypoint CLI:

anypoint.platform.config.analytics.agent.enabled=true